pyproject.toml

```diff
@@ -33,13 +34,16 @@ speedcopy = "^2.1"
 six = "^1.15"
 wsrpc_aiohttp = "^3.1.1" # websocket server
 pywin32 = { version = "300", markers = "sys_platform == 'win32'" }
-jinxed = { version = "^1.0.1", markers = "sys_platform == 'darwin'" }
+jinxed = [
+    { version = "^1.0.1", markers = "sys_platform == 'darwin'" },
+    { version = "^1.0.1", markers = "sys_platform == 'linux'" }
+]
 
 [tool.poetry.dev-dependencies]
 flake8 = "^3.7"
 autopep8 = "^1.4"
 coverage = "*"
-cx_freeze = "^6.1"
+cx_freeze = "=6.2"
```

Comment:
This should be specified as `~6.2` to include subversions. And maybe we should limit it only to darwin and use latest on windows/linux where there's no problem?

Reply:
Wouldn't it be better to keep it consistent? I agree with the looser restriction on the version though. Albeit, they are hardly gonna release a patch for and old version.